src/Vts/Modeling/Spectroscopy/Tissue.cs

```diff
@@ -148,18 +159,50 @@ public OpticalProperties GetOpticalProperties(double wavelength)
         /// <summary>
         /// Returns the optical properties for a given wavelength
         /// </summary>
+        /// <param name="wavelength">Wavelength</param>
+        /// <param name="lambda0">Wavelength normalization factor</param>
+        /// <returns>The optical properties</returns>
+        public OpticalProperties GetOpticalProperties(double wavelength, double lambda0)

```

Comment:
Since GetOpticalProperties is not in the interface, I think you could combine these 2 methods by passing the lambda0 as an optional parameter like we did for GetMusp initially. Then if lambda0 is NaN we call GetMusp without that parameter, otherwise we call it with the parameter.